recipes-scanners/clamav/clamav_1.4.3.bb

```diff
@@ -215,7 +235,7 @@ USERADD_PACKAGES = "${PN}-freshclam"
 GROUPADD_PARAM:${PN}-freshclam = "--system ${CLAMAV_GROUP}"
 USERADD_PARAM:${PN}-freshclam = "--system -g ${CLAMAV_GROUP} --home-dir \
                                  ${localstatedir}/lib/${BPN} \
-                                 --no-create-home --shell /sbin/nologin ${CLAMAV_USER}"
+                                 --create-home --shell /sbin/nologin ${CLAMAV_USER}"
```

Comment:
Home directories should only be created for real human user accounts. Why is this necessary?